app/app.go

```diff
@@ -142,7 +142,7 @@ var (
 	// DefaultNodeHome default home directories for the application daemon
 	DefaultNodeHome string
 	// fee collector account, module accounts and their permissions
-	maccPerms = map[string][]string{
+	MaccPerms = map[string][]string{
```

Comment:
Why is this made public?

Reply:
just to import in a test, I can duplicate the code to avoid exposure

Reply:
rollback to private maccPerms